doc/classes/EditorImportPlugin.xml

```diff
@@ -241,7 +241,7 @@
 				Imports [param source_file] with the import [param options] specified. Should return [constant @GlobalScope.OK] if the import is successful, other values indicate failure.
 				The imported resource is expected to be saved to [code]save_path + "." + _get_save_extension()[/code]. If a different variant is preferred for a [url=$DOCS_URL/tutorials/export/feature_tags.html]feature tag[/url], save the variant to [code]save_path + "." + tag + "." + _get_save_extension()[/code] and add the feature tag to [param platform_variants].
```

Comment:
This big tangent smells like it was added later due to a reported issue. Noting this down for later.